recipes/simbody/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "3.7.0":
+    url: "https://github.com/simbody/simbody/archive/refs/tags/Simbody-3.7.tar.gz"
+    sha256: "d371a92d440991400cb8e8e2473277a75307abb916e5aabc14194bea841b804a"
+
+patches:
+  "3.7.0":
+    - base_path: "source_subfolder"
+      patch_file: "patches/0001-simbody-3.7.0-cmake-fixes.patch"
```

Comment:
```suggestion
      patch_file: "patches/0001-simbody-3.7.0-cmake-fixes.patch"
      description: "Fix Lapack and OpenBLAS target import on CMake"
      patch_type: "backport"
```

Reply:
done

Reply:
are you sure this is correct  ? Hooks resulted in error 
```
 ERROR: [CONANDATA.YML FORMAT (KB-H030)] Additional entries ['description'] not allowed in 'patches':'3.7.0' of conandata.yml (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H030)
```

Reply:
Sorry, it's patch_description: https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches